dataclients/kubernetes/fixtures/skipper_route_crd_complex_example.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: zalando.org/v1
+kind: RouteGroup
+metadata:
+  name: my-routes
+spec:
+  backends:
+  - serviceName: foo-service-v1
+    servicePort: 80
+  paths:
+  - path: /api/resource
+    method: post
+    configs:
+    - filters:
+      - f1("foo")
+      - f2("bar", "baz")
+      predicates:
+      - P1("serviceUser")
+      - P2("bar", "baz")
+    - filters:
+      - f1("dosomething")
+      predicates:
+      - P1("adminusers")
+  - path: /api/resource/*
```

Comment:
Rather than re-declare path over and over again, operations could be listed under a path as in OpenApi:
```
        - path: /api/resource/*
          - operation:  get
            configs:
              - filters:
                  - f("foo")
                predicates:
                  - P("b", 0.4)
                backend:  # overrides default
                  serviceName: migrate-to-bar-service
                  servicePort: 8080
          - operation:  put
            configs:
              - filters:
                  - f("foo")
                predicates:
                  - P("b", 0.4)
                backend:  # overrides default
                  serviceName: migrate-to-bar-service
                  servicePort: 8080
          - operation:  post
            configs:
              - filters:
                  - f("foo")
                predicates:
                  - P("b", 0.4)
                backend:  # overrides default
                  serviceName: migrate-to-bar-service
                  servicePort: 8080
```

Reply:
I like the suggestion.
@aryszka and @aermakov-zalando what do you think? Any opinions on that?

Reply:
If the `method` selector is changed to support multiple values (I think I proposed it in the other discussion), does it actually lead to an improvement? Or is it just because the "path" predicated is required in Skipper?

Reply:
Agree, more DRY and also familiar for the users.